Source/CesiumRuntime/Public/CesiumFeatureIdAttribute.h

```diff
@@ -2,19 +2,31 @@
 
 #pragma once
 
-#include "CesiumFeatureTable.h"
 #include "CesiumGltf/AccessorView.h"
 #include "Kismet/BlueprintFunctionLibrary.h"
 #include "CesiumFeatureIdAttribute.generated.h"
 
 namespace CesiumGltf {
 struct Model;
 struct Accessor;
-struct FeatureTable;
 } // namespace CesiumGltf
 
 /**
- * @brief A blueprint-accessible wrapper for a feature id attribute from a glTF
+ * @brief Reports the status of a FCesiumFeatureIdAttribute. If the feature ID
+ * attribute cannot be accessed, this briefly indicates why.
+ */
+UENUM(BlueprintType)
+enum class ECesiumFeatureIdAttributeStatus : uint8 {
```

Comment:
Are these "status" enums overkill? I thought it might be helpful to expose to the user if something was wrong, but I know that deviates from CesiumJS's "silently fail" philosophy. I could also consolidate it to a bool that indicates plainly whether the thing is valid.

Reply:
This is great! Silently failing (or even failing without explanation) just means more frustrated users and more questions for us on the forums.